modelzoo/WnD/TensorFlow2/data_processing/spark-defaults.conf

```diff
@@ -0,0 +1,10 @@
+spark.eventLog.enabled             true
```

Comment:
Do you need to include Spark config in your own module or leave it to docker? 
and as DLRM, DIEN also uses Spark, DLRM will include this spark default in DLRM/DIEN folder as well? 

@Peach-He  @xuechendi  what do you think?

Reply:
I follow DIEN data preprocess guide for WnD data preprocess. If Spark config is common for WnD, DLRM and DIEN, I think we can keep it just in docker rather in e2eAIOK

Reply:
I found all three spark-defaults.conf are very same, either put in docker or copy during setup is OK. I would suggest to use copy, so that user can know where spark is configured, and can doing change if they need to.

Reply:
make sense, I will maintain spark config file in a common folder and copy the config files when building docker image, so that wnd and dien can use it directly.